ci/build.yml

```diff
@@ -60,7 +60,6 @@ jobs:
 - job: Linux
   dependsOn:
     - check_for_release
-    - git_sha
```

Comment:
The explicit dependency is not used, and the job structure doesn't change because `check_for_release` depends on `git_sha`.